src/marqo_commons/model_registry/model_properties_data/no_model.py

```diff
@@ -0,0 +1,33 @@
+"""
+No-model properties
+
+This file contains random model properties. It is intended to be used in conjunction with the model registry
+and should not be used in isolation.
+"""
+from typing import Dict, List
+
+from marqo_commons.model_registry.model_properties_object import ModelProperties, VectorNumericType, Modality, ModelType
+from marqo_commons.model_registry.utils import convert_model_properties_to_dict
+
+
+class NoModelProperties(ModelProperties):
+    vector_numeric_type: VectorNumericType = VectorNumericType.float32
+    default_memory_size: float = 0.1
+    modality: List[Modality] = [Modality.text, Modality.image]
+    type: ModelType = ModelType.no_model
+    tokens: int
+    notes: str = ""
+
+
+@convert_model_properties_to_dict
+def _get_no_model_properties() -> Dict:
+    NO_MODEL_PROPERTIES = {
+        'no_model': NoModelProperties(
+            name='no_model',
+            dimensions=0,
```

Comment:
Will this 0 dimension bypass the model check? E.g., requires the user to provide a dimension

Reply:
actually I got  this from here
https://github.com/marqo-ai/marqo/blob/bf47a80b423d09f82002fcb6cf03b2649d849023/src/marqo/s2_inference/model_registry.py#L1966-L1972

if `no_model` doesn't make sense I can remove it , regarding the dimension I'm not sure what marqo will do  if dimension is zero

Reply:
ok I can make `dimensions` as optional and not set this field

Reply:
Yeah just checked and setting it to 0 will error out. Thanks for the update!

Reply:
Actually, would it be better if we set it to a required field but only make it optional for this model?

Reply:
All the models have a fix dimension in the model properties except this special one.

Reply:
Nevermind, I think your solution is fine.